### DISCUSSION TITLE
Air purifier recognition problem after upgrading to v1.4.3

### DISCUSSION THREAD

**Discussion by oupeng** [merdok/homebridge-miot#386] [Q&A]
Air purifier function difference
I own three air purifiers, all Xiaomi, before version v1.4.0, it fits homekit perfectly, after experiencing node.js v18 after micloud broken chain, after v1.4.3 reallocation, it can't have the past features, how do I need to configure it?
![172645p9hoawu243ubscf3](https://user-images.githubusercontent.com/3167663/200272420-ef7320a6-4824-4501-aef7-eaaa98bf634b.jpg)
![172652ttuocmoya12z5zui](https://user-images.githubusercontent.com/3167663/200272449-168a3b7b-abea-419a-816e-776f21614925.jpg)
！after！
![172639kdl0q544l80l2q84](https://user-images.githubusercontent.com/3167663/200272544-8de3995b-894e-4226-9128-c9568f17e3c5.jpg)
！now！
thanks

node.js v18.12.1
homebridge v1.4.1
miot v1.4.3
mode: zhimi.airpurifier.v6/zhimi.airpurifier.v6/zhimi.airpurifier.ma2

**Comment by merdok** [comment]
Sorry I do not quite understand what you mean, can you please explain what exactly the problem is more detailed so I can understand?
